mk_verificator/tests/ceph/test_ceph.py

```diff
@@ -0,0 +1,11 @@
+from mk_verificator import utils
+
+
+def test_check_ceph_osd(local_salt_client):
+    config = utils.get_configuration(__file__)
+    osd_fail = \
```

Comment:
could you please share what salt returns? (you know, unfortunately I have no ceph under my hand =( )